Source/Core/Core/HW/EXI_DeviceSlippi.cpp

```diff
@@ -523,10 +527,35 @@ void CEXISlippi::createNewFile()
 		closeFile();
 	}
 
-	std::string dirpath = File::GetExeDirectory();
-	File::CreateDir(dirpath + DIR_SEP + "Slippi");
-	std::string filepath = dirpath + DIR_SEP + generateFileName();
+	std::string dirpath = SConfig::GetInstance().m_strSlippiReplayDir;
+
+	// First, ensure that the root Slippi replay directory is created
+	File::CreateDir(dirpath);
```

Comment:
It would be better to use `File::CreateFullPath` here, since it can handle several levels of directories needing to be created.

Reply:
Good point. I missed this function.